.gitlab-ci.yml

```diff
@@ -17,7 +17,13 @@ stages:
 trigger-agent-build:
   rules:
     - *skip_stable_branches
-    - when: always
+    - changes:
+        - "**/*.rb"
+        - Rakefile
+        - omnibus-software.gemspec
+        - Gemfile
```

Comment:
Could you move this in an anchor to avoid duplicating it throughout the file?

Reply:
I'd add a comment mentioning that the list of change must be the last in the list, otherwise they will always evaluate to true in scheduled pipelines, which we don't want